Changes

```diff
@@ -1,6 +1,42 @@
 Release history for Zonemaster component Zonemaster-Backend
 
 
+v10.0.0 2022-12-19 (public release version)
+
+ [Deprecation]
+ - The use of language tags that includes country code is
+   deprecated. See "docs/API.md#language-tag". Such
+   language codes will be made illegal in the v2023.1 release.
+   Use language codes without country code instead.
+ - The use of an empty string in the "LANGUAGE.locale"
+   setting is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+ - The use of two or more locale tags with the same language code
+   in the configuration is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+
+ [Breaking changes]
+ - Removes database primary key from API method "get_test_results"
+   (#946, #949).
+
+ [Features]
+ - Provides human readable messages when the test agent dies (#1058)
+ - Adds testcase results (#1055)
+ - Adds Zonemaster-LDNS version to API method "version_info" (#1050)
+
+ [Fixes]
+ - Updates deprecation information in "docs/API.pm" and
+   "docs/Configuration.md" (#1067)
+ - Corrects the license statement in Backend.pm (#1059)
+ - Fixes recursor init (#1057)
```

Comment:
We should use the changes file to describe the user perspective, not the code change. I suggest updating the text of this item to `Fix get_data_from_parent_zone method.`.

Reply:
I try to do that. In this case I was too lazy to try to find out what that would be. Fixed now.